docs/howtos/layers/points.md

```diff
@@ -435,13 +436,13 @@ nbscreenshot(viewer, alt_text="3 points overlaid on an astronaut image, where th
 viewer.close()
 ```
 
-In the example above, the `point_properties` were provided as a dictionary with
-two properties: `good_point` and `confidence`. The values of each property are
-stored in a NumPy ndarray with length 3 since there were 3 coordinates provided
-in `points`. We set the face color as a function of the `confidence` property by
-providing the keyword argument `face_color='confidence'` to the
-`viewer.add_points()` method. We set the colormap to viridis using the
-`face_colormap` keyword argument as `face_colormap='viridis'`.
+In the example above, the `point_features` were provided as a
+dictionary with two feature columns: `good_point` and `confidence`.
+The table has three rows since there were three coordinates provided in `points`.
+We set the face color as a function of the `confidence` feature by providing the
```

Comment:
maybe a bit confusing for folks unfamiliar with pandas to talk about columns/rows in a dictionary.

Reply:
Ah, indeed, maybe worth saying that this is converted internally to a data frame with two columns and three rows? (I wouldn't specify pandas necessarily, but maybe.)

Reply:
Agreed this paragraph is mixing things a bit too much. Let me try to fix that.

But I also want to make a distinction between the meaning of `features` and the type used to implement it. Ideally, we could do that in one place for all layers (see https://github.com/napari/docs/issues/23).

For now, there's a little effort above to describe what the table is in the "### Using the points features table" section. I'll clean that up a little and will mention that the table can be provided as a dictionary where the values are array-likes of the same length.

Reply:
I also (tried to) put a link back to the section that describes the dictionary and table/dataframe relationship in both of these similar paragraphs.